tracing-appender/src/file_appender.rs

```diff
@@ -36,6 +45,10 @@ impl FileAppender {
     pub fn get_writer(self) -> FileWriter {
         self.log_writer
     }
+
+    pub fn get_error_counter(self) -> Arc<AtomicU64> {
+        self.error_counter.clone()
+    }
```

Comment:
Nit: prefixing accessor methods with `get_` [isn't considered idiomatic in Rust](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#getter/setter-methods-[rfc-344]), so these should probably be 
```suggestion
    pub fn writer(self) -> FileWriter {
        self.log_writer
    }

    pub fn error_counter(self) -> Arc<AtomicU64> {
        self.error_counter.clone()
    }
```